src/main.rs

```diff
@@ -83,7 +83,9 @@ fn run() -> Result<(), ContextualError> {
         && miniserve_config
             .path
             .symlink_metadata()
-            .map_err(|e| ContextualError::IOError("Failed to retrieve symlink's metadata".to_string(), e))?
+            .map_err(|e| {
+                ContextualError::IOError("Failed to retrieve symlink's metadata".to_string(), e)
+            })?
```

Comment:
You just split this line into 3, is this necessary?

Reply:
rustfmt auto split line to 3

Reply:
It seems that you need to undo that to please Clippy

https://travis-ci.org/svenstaro/miniserve/jobs/538610810